inc/field.class.php

```diff
@@ -750,8 +750,45 @@ public static function showForTab($params)
             return false;
         }
 
+        //JS to trigger any change and check if container need to be display or not
+        $ajax_url   = Plugin::getWebDir('fields') . '/ajax/container_display_condition.php';
+        $items_id = $item->getID() ? $item->getID() : 0;
+        echo Html::scriptBlock(<<<JAVASCRIPT
+
+            function checkContainerVisibility(){
+                var data = $('#itil-form').serializeArray().reduce(function(obj, item) {
```

Comment:
`#itil-form` will make this only work on ITIL objects. Many other forms can be targetted by `.asset_form`, but, in fact, we cannnot be sure that all forms could be targetted this way.
Maybe the selector should be `$('#fields_container').closest('form')`.

Also, having a random value appended to identifier may prevent issues on pages that may display multiple forms (ITIL timeline for instance).